src/lib/bootloader/systeminfo.rb

```diff
@@ -108,6 +118,31 @@ def s390_secure_boot_active?
       rescue StandardError
         false
       end
+
+      # The partition where zipl is installed.
+      #
+      # @return [Y2Storage::Partition, NilClass] zipl partition
+      def zipl_device
+        staging = Y2Storage::StorageManager.instance.staging
+        mountpoint = Y2Storage::MountPoint.find_by_path(staging, "/boot/zipl").first
```

Comment:
does it found if e.g. whole /boot is ext? and so no specific mp for zipl?

Reply:
Hm, good point. I was assuming /boot/zipl is always separate.